GPSCoord location
float64 radius